src/User/AvatarUploader.php

```diff
@@ -46,7 +46,7 @@ public function upload(User $user, Image $image)
 
     public function remove(User $user)
     {
-        $avatarPath = $user->avatar_path;
+        $avatarPath = array_get($user->getAttributes(), 'avatar_url');
```

Comment:
Please import the Laravel Arr class and use `Arr::get()`.

Also this method might need a check whether the url has a protocol or not. There is a difference between `https://somestorage.com/avatar.png` vs `/assets/avatars/avatar.png`.

Reply:
@luceos do you think the `if ($this->uploadDir->has($avatarPath))` check will be sufficient?

Reply:
My thought exactly @tobscure, I am not entirely sure though. What do you think @franzliedke?

Reply:
@luceos Already changed to Arr::get. So is it not preferred in flarum project to use functions without namespaces?

Reply:
Most Laravel helpers are deprecated in the framework itself. It won't make sense to keep them around as we'd have to refactor them in the future anyway.